components/composer/src/Composer.svelte

```diff
@@ -228,6 +225,10 @@
       fileSelector.value = "";
       if (beforeFileUpload) beforeFileUpload(file);
 
+      if (file.size >= 5000000) {
```

Comment:
This should either be `if (file.size >= 4000000) {` or the docs should be updated because it says the [limit is 4 mB](https://developer.nylas.com/docs/user-experience/components/composer-component/#attachments)

Reply:
Ou thanks Heather! I tried a file with 4.6MB and it went through though... but I think keeping it to the Doc's standard is better. 
```suggestion
      if (file.size >= 4000000) {
```